x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -374,6 +377,70 @@ export const uiSettings: Record<string, UiSettings> = {
     schema: schema.boolean(),
     requiresPageReload: false,
   },
+  [profilingPerCoreWatt]: {
+    category: [observabilityFeatureId],
+    name: i18n.translate('xpack.observability.profilingPerCoreWattUiSettingName', {
+      defaultMessage: 'Per Core Watts',
+    }),
+    value: 7,
+    description: i18n.translate('xpack.observability.profilingPerCoreWattUiSettingDescription', {
+      defaultMessage: `The average amortized per-core power consumption (based on 100% CPU utilization).`,
```

Comment:
Do we want to say why it defaults to 7 or why the user might want to change it?

Reply:
@iogbole 👆🏻